mkt/developers/urls.py

```diff
@@ -210,6 +211,10 @@ def provider_patterns(prefix):
     '',
     url(r'^apps/app/(?P<pk>[^/<>"\']+)/content-ratings/pingback/',
         ContentRatingsPingback.as_view(), name='content-ratings-pingback'),
+    # IARC v2 pingback does not use the app pk, instead it uses IARCRequest
+    # uuid, but sends it as POST data.
+    url(r'^apps/iarc/pingback/', ContentRatingsPingbackV2.as_view(),
```

Comment:
Maybe yeah. Or just `/iarc/pushcert/` ? I wanted to avoid the "v2" in the URL because it was confusing with our own API version number